db/schema.rb

```diff
@@ -259,7 +268,9 @@
     t.date "last_decay", default: -> { "now()" }
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.string "values", default: [], array: true
     t.index ["user_id", "counter_name"], name: "index_user_counters_on_user_id_and_counter_name", unique: true
+    t.index ["values"], name: "index_user_counters_on_values", using: :gin
```

Comment:
good catch Mark, it's from this PR: https://github.com/CruGlobal/mobile-content-api/pull/826

which we've put on indefinite hold because we have no actual use cases that require it

Reply:
Andrew, that must have snuck into your local db; you should revert it.